docs/theory/Validation_pipe_diameter_sizing.rst

```diff
@@ -0,0 +1,1148 @@
+
+The optimization routine determines what is the "optimal" diameter for a certain pipe.
+The optimal solution depends on many parameters.
+Also, not all properties are the same in practice if you only vary the diameter.
+Think about the thickness of the insulation that typically scales with the pipe diameter.
+
+The current optimizer routine optimizes the diameter of pipes based on pipe properties from the `ESDL Energy Data Repository (EDR) <https://edr.hesi.energy/cat/Assets>`_.
+Currently, the optimizer routine only supports for the Steel-S1 class from the `Logstor catalog <https://www.logstor.com/media/6506/product-catalogue-uk-202003.pdf>`_. It contains pipe properties for bonded pipe systems of district heating pipes
+with insulation series 1.  
+These pipes are steel pipes with a PUR layer for insulation and a PE casing.
+
+
+.. improve: allow user to specify insulation class: 1, 2 or 3.
+
+The library contains the following properties for several DN sizes:
+
+* **name**: Type of material inner pipe - insulation class - DN size.
+* **inner_diameter**: inner diameter of the pipe [m].
+* **u_1**: thermal transmittance (i.e., U-value) 1 [Wm\ :sup:`-2` K\ :sup:`-1`].
+* **u_2**: thermal transmittance (i.e., U-value) 2 [Wm\ :sup:`-2` K\ :sup:`-1`].
+* **insulation_thicknesses**: ordered list (from inner layer outwards) of thicknesses of the insulation layers in [m].
+* **conductivities_insulation**: ordered list (from inner layer outwards) of conductivity in [W/mK].
+* **investment_costs**: combined installation and investment cost [€/m].
+
+The optimizer routine assumes that the conductivity values of the insulation layers do not depend on temperature.
+In general, the conductivity for PUR increases over time. This is not taken in account but the lifetime average is assumed.
+
+The properties for the different insulation classes can be found in Chapter :ref:`chp_logstor`.
+
+.. _subsec_V_Network_temperatures:
+
+Network temperatures
+^^^^^^^^^^^^^^^^^^^^
+
+Regarding network temperatures (supply and return) the optimizer routine assumes the following:
+
+* The user provides network temperatures as input variables by setting one "carrier" with a certain temperature to all supply pipes and one "carrier" with a certain temperature to all return pipes. 
+* A decoupled hydraulic network has only one supply and one return temperature. Thus it is not possible that different sources supply flow to the network at different temperatures.
+* The supply carrier and return carriers' temperatures are fixed throughout the optimization.  
+* Calculations are done for the relative heat between the supply and return lines. This relative heat is only transported over the supply line.
+* The network temperature should be in the range 20 to 130 :math:`^\circ C`.
+
+.. The default temperature is 0.0?
+
+.. _subsec_V_Flow:
+
+Flow
+^^^^
+
+Regarding flows, the optimizer routine assumes the following:
+
+* Minimum flow velocity, :math:`v_{min}`, is 0.0 m/s.
```

Comment:
Actually, the minimum value is defined at several places in the code. In heat_mixin it is 0.005, In class EndScenarioSizing (grow_workflow.py), it is 0.001 and EndScenarioSizingStaged it is 0.0. Since the routine uses EndScenarioSizingStagedHIGHS(EndScenarioSizingStaged), it is 0.0 to me.

Reply:
Good point!